charts/gsp-cluster/templates/00-aws-auth/sre-cluster-role.yaml

```diff
@@ -6,7 +6,24 @@ metadata:
   name: sre
 rules:
   - apiGroups: [""]
-    resources: ["pods"]
+    resources:
+    - configmaps
+    - pods
+    - secrets
+    - services
```

Comment:
not in the email

Reply:
I spoke with Howard about this and it's OK...I just forgot to add it to the email. It should be included in the ADR though.